### PR TITLE
Reworked how matcap view is triggered to avoid having it in game view when scene lighting toggle is off

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
Tried quite a few things, but this is the least intrusive way. 

Fix for https://fogbugz.unity3d.com/f/cases/1156102/

### CODE REVIEW

**Comment by FrancescoC-unity** [comment]
A problem with exposure handling has been found, will investigate monday

**Comment by FrancescoC-unity** [comment]
This should now work, could you try it again @JMargevics? Thanks

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs [hunk 1]
```diff
@@ -3299,7 +3301,7 @@ static void RenderComputeAsPixelDeferredLighting(in DeferredLightingParameters p
             }
             else
             {
-                CoreUtils.SetKeyword(cmd, "DEBUG_DISPLAY", parameters.debugDisplaySettings.IsDebugDisplayEnabled());
+                CoreUtils.SetKeyword(cmd, "DEBUG_DISPLAY", true/* parameters.debugDisplaySettings.IsDebugDisplayEnabled()*/);
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs]: Is this intended?

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs]: Nope, debug code, good catch

**Pull request merged**